### PR TITLE
Make a custom HACS repo and add some validation

### PR DESCRIPTION
**Pull request by jmtatsch**

### CODE REVIEW

**Comment by jmtatsch** [comment]
Any chance for this to get reviewed/merged?

**Comment by cytech** [comment]
when i get some time to learn about HACS :)
thanx

**Comment by jmtatsch** [comment]
@cytech do you need some help maintaining this repo, reviewing pull requests etc?

**Comment by rudizl** [comment]
@cytech - this will be very useful for all of us.

File: custom_components/wundergroundpws/manifest.json [hunk 1]
```diff
@@ -6,5 +6,6 @@
   "issue_tracker": "https://github.com/cytech/Home-Assistant-wundergroundpws/issues/",
   "requirements": [],
   "dependencies": [],
-  "codeowners": ["@cytech"]
+  "codeowners": ["@cytech"],
+   "iot_class": "cloud_polling"
```
Comment (shtrom) [inline custom_components/wundergroundpws/manifest.json:10]:
```suggestion
  "iot_class": "cloud_polling"
```

**Review by shtrom — COMMENTED** [review]
Looks good to me, though I'm not sure if the codeql Github Workflow is strictly necessary.

FWIW, this integration already works well with HACS when the reop is added manually. So my understanding is that this only support the auto-detection of the repo by HACS the rest of the time.

**Comment by HrGaertner** [comment]
Would it be possible to merge this request? That would be really great! Thanks!

**Pull request closed without merging**